demos/emission_models/thermal_charge_exchange.py

```diff
@@ -0,0 +1,130 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.constants import electron_mass, atomic_mass
+
+from raysect.core import translate, rotate_basis, Point3D, Vector3D
+from raysect.primitive import Box
+from raysect.optical import World, Ray
+
+from cherab.core import Plasma, Beam, Maxwellian, Species
+from cherab.core.math import ScalarToVectorFunction3D
+from cherab.core.atomic import hydrogen, deuterium, carbon, Line
+from cherab.core.model import SingleRayAttenuator, BeamCXLine, ThermalCXLine
+from cherab.tools.plasmas.slab import NeutralFunction, IonFunction
+from cherab.openadas import OpenADAS
+from cherab.openadas.install import install_adf15
+
+
+# Install PECs for CVI spectral lines
+install_adf15(carbon, 5, 'adf15/pec96#c/pec96#c_pju#c5.dat', download=True)
```

Comment:
I feel a bit uncomfortable about a demo modifying the users' atomic data repository. Should we in fact add this file to `repository.populate()`? It's unclear why we don't have more carbon lines in there by default, given they parse fine with the existing parser? @Mateasek any thoughts?

Reply:
Good catch, I think user database shouldn't be changed from a demo. It could lead to errors or worse, for example due to filesystem rights restrictions. If the file is provided by Open ADAS, I agree it should be rather added to the `repository.populate()`. If it is not available yet, I would say comment out the file installation and add an additional comment for the users that they have to install it this way.

Reply:
Agree, I added this to `populate()`.